date.go

```diff
@@ -241,3 +241,53 @@ func Min(d1, d2 Date) Date {
 	}
 	return d2
 }
+
+// NullDate represents a nullable date with compatibility for
+// scanning null values from the database.
+type NullDate struct {
+	Date  Date
+	Valid bool
+}
+
+func NewNullDate(d Date) NullDate {
+	return NullDate{
+		Date:  d,
+		Valid: true,
+	}
+}
+
+// Scan implements the sql.Scanner interface for database deserialization.
+func (d *NullDate) Scan(value interface{}) error {
```

Comment:
The practice now is to use `any` over `interface{}` for types. Since any is an alias for interface it should still adhere to the `sql.Scanner` and `driver.Valuer` interfaces.

Reply:
So this is technically a public library, though ours is branched off, so I think that is why the version was set so low on it. I think that is why `any` wasn't being used. In the end though this is our version, so I don't see an issue updating the version and using any. So I have updated this too.